typed_python/compiler/type_wrappers/arithmetic_wrapper.py

```diff
@@ -328,12 +328,20 @@ def convert_builtin(self, f, context, expr, a1=None):
 
     def convert_unary_op(self, context, left, op):
         if op.matches.Not:
+            if left.isConstant:
+                return context.constant(not left.constantValue)
             return context.pushPod(bool, left.nonref_expr.logical_not())
         if op.matches.Invert:
+            if left.isConstant:
+                return context.constant(~left.constantValue)
             return context.pushPod(self, left.nonref_expr.bitwise_not())
         if op.matches.USub:
+            if left.isConstant:
+                return context.constant(-left.constantValue)
             return context.pushPod(self, left.nonref_expr.negate())
         if op.matches.UAdd:
+            if left.isConstant:
+                return context.constant(left.constatValue)
```

Comment:
can we try to get tests that exercise this code? because 'constatValue' won't work...

Reply:
Added tests to check all the unary ops on constant ints, floats, bools are behaving as expected (and fixed the typo)